docs/marionette.view.md

```diff
@@ -333,9 +333,9 @@ bound at the time of instantiation, and an exception will be thrown
 if the handlers on the view do not exist.
 
 The `modelEvents` and `collectionEvents` will be bound and
-unbound with the Backbone.View `delegateEvents` and `undelegateEvents`
-method calls. This allows the view to be re-used and have
-the model and collection events re-bound.
+unbound with the Backbone.View `delegateEntityEvents` and `undelegateEntityEvents`
+method calls. `delegateEntityEvents` is called in the View's `constructor` and
+entity events are unbound during the View's `destroy`.
```

Comment:
I guess I'm not sure of the issue you're addressing.
`modelEvents` / `collectionEvents` will only be bound in the `constructor`
and they're only removed when the view is destroyed.
If the view has been `preventDestroy`ed, they'll still be listening.

Are you saying they should stop listening if destroy is prevented?